src/components/DialogFormButton.js

```diff
@@ -18,24 +23,22 @@ export default function DialogFormButton({
   disabled
 }) {
   const [open, setOpen] = useState(false)
-  const [value, setValue] = useState('')
-
-  const handleClickOpen = () => {
-    setOpen(true)
-  }
+  const [value, setValue] = useState()
 
-  const handleClose = () => {
+  const reset = useCallback(() => {
     setOpen(false)
-  }
+    setValue()
+  }, [])
+
+  const handleClickOpen = () => setOpen(true)
+  const handleClose = () => reset()
 
   const handleConfirm = () => {
     onConfirm(value)
-    setOpen(false)
+    reset()
```

Comment:
I think we should use `useCallback` here otherwise it doesn't make sense to use it in the reset function definition